internal/provider/resource_hvn_test.go

```diff
@@ -102,6 +106,41 @@ func testAccCheckHvnExists(name string) resource.TestCheckFunc {
 	}
 }
 
+func testSelfLink(name string, expectedID, expectedType string) resource.TestCheckFunc {
```

Comment:
thanks for adding this! probably something we could share with future tests :)

Reply:
yes, this is how I saw it too